### DISCUSSION TITLE
Podman Desktop supporting Options

### DISCUSSION THREAD

**Discussion by jonahlytlevoya** [podman-desktop/podman-desktop#5266] [General]
Hey, Does Podman Desktop support adding options to Container/Image creation? Is this something that may be supported in the future?

**Comment by benoitf** [comment]
hello @jonahlytlevoya yes it should be supported like we do support that for running containers

please file an enhancement issue (I think there was one but I'm not able to find it)

**Reply by jonahlytlevoya** [reply]
Great, as in it is already supported or will be in the future?

**Reply by benoitf** [reply]
currently supported for run, not for build

**Reply by benoitf** [reply]
but I don't see why we could not add support on build part
